src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -223,13 +222,17 @@ public static void setJDBCLoggingDirectory() {
    *
    * @param config connector configuration
    */
+  // TODO @rcheng: question - can i change this exception to also return the invalid configs?
```

Comment:
validateProxySetting is only ever called within a try/catch. It would be more performant and easier to log if we  returned a list of invalid configs. Can I make this change?